man/man1/ztest.1

```diff
@@ -147,6 +147,12 @@ ztest -f / -V -T 120
 
 .SH "ENVIRONMENT VARIABLES"
 .TP
+.B "ZFS_HOSTID=id"
+Use \fBid\fR instead of the SPL hostid to identify this host.  Intended for use
+with ztest, but this environment variable will affect any utility which uses
+libzpool, including \fBzpool(8)\fR.  Since the kernel is unaware of this setting
```

Comment:
Yes, it is.  Perhaps @behlendorf can answer the why part; that decision predates me.  This does have the effect of ensuring that the libzfs code works, and reduces duplicated code in the utilities.